### ISSUE TITLE
Hidden column causing exceptions since 5.6.0 

### ISSUE THREAD

**Issue by robrez** [vaadin/vaadin-grid#1729]
### Description

Receiving errors since 5.6.0

```bash
Uncaught TypeError: Cannot read property '_content' of undefined
    at HTMLElement.ColumnBaseMixin.__runRenderer (vaadin-grid-column.js:242)
    at HTMLElement.ColumnBaseMixin._renderHeaderAndFooter (vaadin-grid-column.js:235)
    at eval (vaadin-grid.js:771)
    at Array.forEach (<anonymous>)
    at eval (vaadin-grid.js:771)
    at Array.forEach (<anonymous>)
    at HTMLElement.GridElement.render (vaadin-grid.js:770)
```

### Live Demo

https://stackblitz.com/edit/typescript-ybspyn

### Steps to reproduce

1. Have a column start out as `hidden`
2. Manually run `grid.render()`

In the repro above, I had made the connection to rendering ... but the repro is over-complicated - it copied a v-grid selection demo because that was closer to my production code.  It appears all that is necessary is just running `render`

![grid-5 6 0-hidden-col](https://user-images.githubusercontent.com/1993389/81127035-9c1bde00-8f02-11ea-9df8-a2305f94a603.gif)

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge
- [x] IE 11
- [x] iOS Safari
- [x] Android Chrome

**Comment by robrez** [comment]
Will `5.6.x` be tagged, or, is the plan to wait on `5.7.0`?

**Comment by web-padawan** [comment]
@robrez sorry for the delay, submitted a PR to cherry-pick: https://github.com/vaadin/vaadin-grid/pull/1736

**Comment by web-padawan** [comment]
Released `5.6.2` with a fix.
